django/contrib/contenttypes/management/commands/remove_stale_contenttypes.py

```diff
@@ -14,6 +14,10 @@ def add_arguments(self, parser):
             '--noinput', '--no-input', action='store_false', dest='interactive',
             help='Tells Django to NOT prompt the user for input of any kind.',
         )
+        parser.add_argument(
```

Comment:
I'm not sure about the name for this argument. Please let me know if there's a better name for this
Thanks!